docs/source/config_syntax.md

```diff
@@ -164,6 +164,7 @@ python -m monai.bundle COMMANDS
 where `COMMANDS` is one of the following: `run`, `verify_metadata`, `ckpt_export`, ...
 (please see `python -m monai.bundle --help` for a list of available options).
 
+The commands support many fexible use cases, like: override config at runtime, predefine CLI args in a file, etc.
```

Comment:
```suggestion
The CLI supports flexible use cases, such as overriding configs at runtime and predefining arguments in a file.
```

Reply:
updated in the latest commit.

Thanks.